howto/trigger-app-with-input-binding/README.md

```diff
@@ -20,7 +20,7 @@ An input binding represents an event resource that Dapr uses to read events from
 
 For the purpose of this HowTo, we'll use a Kafka binding. You can find a list of the different binding specs [here](../../reference/specs/bindings/README.md).
 
-Create the following YAML file, named binding.yaml, and save this to the /components sub-folder in your application directory:
+Create the following YAML file, named binding.yaml, and save this to a `components` sub-folder in your application directory:
```

Comment:
also shouldn't the path be specified with the flag? Otherwise only the default `components` folder will be used.

Reply:
added clarification.